test/linear_method_tests.jl

```diff
@@ -30,5 +30,37 @@ function (p::typeof(L))(::Type{Val{:analytic}},t,u0)
 end
 
 prob = ODEProblem(L,u0,(0.0,1.0))
-sol = solve(prob,StrangSplitting(),dt=1/10)
+sol = solve(prob,MidpointSplitting(),dt=1/10)
 using Plots; pyplot; plot(sol)
```

Comment:
Can't plot in tests because it'll bog down the CI which is why no plotting package is a dependency. This'll need to be commented out.